### ISSUE TITLE
Asset 503: make `UniProtKB:P00734` the expected output ID/change the output name

### ISSUE THREAD

**Issue by colleenXu** [NCATSTranslator/Tests#83]
[Asset 503](https://github.com/NCATSTranslator/Tests/blob/main/test_assets/Asset_503.json) says the expected output name is `UNIPROTKB:P00734`, which is the gene F2 + its proteins ([Dev NodeNorm](https://nodenormalization-sri.renci.org/get_normalized_nodes?curie=UNIPROTKB:P00734&conflate=true&drug_chemical_conflate=true&description=false)). 

However, the output ID `CHEMBL.TARGET:CHEMBL204` is likely something no ARA/KP recognizes. [NodeNorm doesn't resolve this ID.](https://nodenormalization-sri.renci.org/get_normalized_nodes?curie=CHEMBL.TARGET:CHEMBL204&conflate=true&drug_chemical_conflate=true&description=false).

I imagine the expected output ID needs to be changed to `UniProtKB:P00734` (correct CURIE format) and the name changed to "F2/thrombin" or something like that.

**Comment by colleenXu** [comment]
I think all tools have been consistently failing this test due to the answer ID. This asset is specifically described in https://github.com/NCATSTranslator/Tests/issues/69

**Comment by maximusunc** [comment]
Putting `UNIPROTKB:P00734` into Name Resolver gives `PR:000051041` which seems to be a subclass of thrombin. And using conflation in NN still gives the same ID. Are you using a different conflation?

**Comment by colleenXu** [comment]
I don't agree with using prothrombin N-glycosylated 4 (human) / [PR:000051041](https://proconsortium.org/cgi-bin/entry_pro?id=PR%3A000051041&retrieve.x=5&retrieve.y=7) as the CURIE. That looks like a very specific variation of prothrombin (N-glycosylated at Asn121) [with very little literature on it](https://glygen.org/protein/P00734#Glycosylation).  

---

F2 gene encodes the protein prothrombin, which is cleaved to create [thrombin](https://en.wikipedia.org/wiki/Thrombin). [UniProtKB:P00734](https://www.uniprot.org/uniprotkb/P00734/entry) is a ID for the F2 gene/prothrombin.

I was using NodeNorm Dev w/ conflation and found that [UniProtKB:P00734](https://nodenormalization-sri.renci.org/get_normalized_nodes?curie=UniProtKB:P00734&conflate=true&drug_chemical_conflate=true&description=false) resolves to a F2/prothrombin entity. 

I think that's an acceptable answer, but I'm not sure what the original writer of this test wanted.

**Comment by sandrine-muller-research** [comment]
please have a look at my response to #84 this is the way I worked with the tests assets: since I am testing the UI I report what the UI states. I think One asset can be a pass or a fail for several reasons (I used to express them but I am not sure the metadata was kept in the long run). Getting what the UI states is helpful to use those test assets for other suites (e.g. a normalization suite).

**Comment by sandrine-muller-research** [comment]
Please assign this email address/GitHub account in the future, I do not receive any notification with the other one. Thanks!

**Comment by sandrine-muller-research** [comment]
FWIW, Please note that currently NameRes does not allow the user to conflate or not. F2, prothrombin, and thrombin are currently in the same clique, which is what is expected for now. This may need to change if we need to unconflate though.
